src/parser.ts

```diff
@@ -88,42 +91,70 @@ export class Parser {
     }
 
     parseLaTeX(log: string) {
-        log = log.replace(/(.{78}(\w|\s|\d|\\|\/))(\r\n|\n)/g, '$1')
         this.buildLogRaw = log
-        const lines = log.replace(/(\r\n)|\r/g, '\n').split('\n')
+        const lines = log.split('\n')
         this.buildLog = []
+
+        let remainingEmptyLines = 0
```

Comment:
I checked the code, and found that `remainingEmptyLines` only has two possible values: 0 and 1. Is it very familiar?

What about changing the variable to `let reachedEmptyLine = false` to make the statement more intuitive?